services/supervisor/src/api.rs

```diff
@@ -59,6 +75,25 @@ impl PrivateApi for ApiImpl<'_> {
     fn start_service(&self, service: StartService) -> Result<Hash, Self::Error> {
         self.broadcast_transaction(service).map_err(From::from)
     }
+
+    fn confirm_config(&self, vote: ConfigVote) -> Result<Hash, Self::Error> {
+        self.broadcast_transaction(vote).map_err(From::from)
+    }
+}
+
+impl PublicApi for ApiImpl<'_> {
+    type Error = api::Error;
+
+    fn consensus_config(&self) -> Result<ConsensusConfig, Self::Error> {
```

Comment:
What about service configs?

Reply:
It seems that the concrete service have to care about it by itself. Exonum doesn't store this information now.

Reply:
OK then.